### PR TITLE
Optional docker image pull.

### PR DESCRIPTION
**Pull request by lukacu**
This PR makes force argument to docker image_present state parameterized from pillar config.

### CODE REVIEW

File: docker/containers.sls [hunk 1]
```diff
@@ -121,7 +121,7 @@ for container, cfg in pillar('docker:containers', {}).items():
         '%s-image' % container,
         name="%s:%s" % (cfg['image'], cfg.get('tag', 'latest')),
         # Pull image even if already present.
-        force=True,
+        force=cfg.get('pull', True),
```
Comment (kostko) [inline docker/containers.sls]: Perhaps change the name to `always_pull`?

Comment (mitar) [inline docker/containers.sls]: Or `refresh`, to be similar to other states. :-)

Comment (mitar) [inline docker/containers.sls]: (But I prefer `always_pull`.)

**Comment by lukacu** [comment]
Hm apparently git managed to merge both branches together so you are free to close the issue and add this small change yourself. Not worth the effort for one line of code :(

**Comment by mitar** [comment]
`git` managed to do it? ;-)

**Comment by mitar** [comment]
I am OK with this being merged as it is.

**Pull request merged**